cli/dxapplet/src/code.sh

```diff
@@ -153,6 +155,10 @@ main() {
             FlameGraph/stackcollapse-perf.pl < perf_genotype > out/perf/genotype.stacks
             /bin/rm -f perf.data
         fi
+
+        # compress the discovered alleles file
+        $compressor -c out/dsals/discovered_alleles.yaml > out/dsals/discovered_alleles.yaml.${compress_ext}
```

Comment:
make the output filename "out/dsals/${output_name}.discovered_alleles.yml.${compress_ext}" (include the quotes in case output_name has spaces for some reason)